Frontend/RecRock-Calendar-Frontend/src/app/calendar/components/calendar/calendar.component.ts

```diff
@@ -24,20 +25,57 @@ export class CalendarComponent implements OnInit {
 
   public viewDate: Date = new Date();
   public events: Event[] = [];
+  public usersInfo: { [key: string]: User } = {};
 
   private user!: User;
 
-  constructor(public readonly dialog: MatDialog, private readonly userService: UserService) {}
+  constructor(
+    public readonly dialog: MatDialog,
+    private readonly userService: UserService,
+    private readonly eventService: EventService
+  ) {}
 
   ngOnInit(): void {
-      let userId= sessionStorage.getItem('userId');
-      if (userId !== null) {
-        this.userService
-        .getById(userId)
-        .then((value) => {
-          this.user = value
+    let userId = sessionStorage.getItem('userId');
```

Comment:
I would create a new file, for example, storage.constans.ts in the root of the project, and I would declare the 'userId' string as a constant because for sure it will be used in multiple places